Traducir.Core/Services/SOStringService.cs

```diff
@@ -596,5 +597,30 @@ Update Strings
             }
             return _strings;
         }
+
+        public async Task<bool> DeleteSuggestionAsync(SOStringSuggestion suggestion)
+        {
+            using (var db = _dbService.GetConnection())
+            {
+                await db.ExecuteAsync(@"
+Update StringSuggestions
+Set StateId = {=DeletedByOwner}
+Where Id = @suggestionId;
+
+Insert Into StringSuggestionHistory
+            (StringSuggestionId, HistoryTypeId, UserId, Comment, CreationDate)
+Values      (@suggestionID, {=DeletedByOwner}, @userId, @comment, @now);",
+                new
+                {
+                    suggestionId = suggestion.Id,
+                    StringSuggestionState.DeletedByOwner,
+                    userId = suggestion.CreatedById,
+                    comment = "Deleted by user",
+                    now = DateTime.UtcNow
+                });
+            }
+            await RefreshCacheAsync();
```

Comment:
don't refresh the entire cache (this forces the load of the 8000+ strings and their suggestions)... instead, pass the `stringId` so that only that string (and its suggestions) are fetched from the db